### PR TITLE
adding hack/verify-boilerplate.sh to make ci target

### PR DESCRIPTION
**Pull request by chrislovecnm**
checking that headers are correct!  BTW I have a PR in for fixing some of the headers.  Until that PR is in, this build should fail.

closes: https://github.com/kubernetes/kops/issues/734

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kops/1044)

### CODE REVIEW

**Comment by chrislovecnm** [comment]
PR that will cause this build to fail is https://github.com/kubernetes/kops/pull/1042

**Comment by chrislovecnm** [comment]
Yep the build is failing as I expected.  It is alive!!

```
sh -c hack/verify-boilerplate.sh
Boilerplate header is wrong for: /home/travis/gopath/src/k8s.io/kops/dns-controller/pkg/dns/record_test.go
Boilerplate header is wrong for: /home/travis/gopath/src/k8s.io/kops/dns-controller/pkg/dns/zonespec_test.go
Boilerplate header is wrong for: /home/travis/gopath/src/k8s.io/kops/dns-controller/pkg/util/equality_test.go
Execute hack/update-header.sh to update headers
make: *** [ci] Error 1
```

**Comment by chrislovecnm** [comment]
Let me rebase and this should pass ci

**Comment by chrislovecnm** [comment]
Good to go now.

File: Makefile [hunk 1]
```diff
@@ -228,8 +224,7 @@ govet:
 # Continuous integration targets
 
 ci: kops nodeup-gocode examples test govet
-	echo "Done"
-
+	sh -c hack/verify-boilerplate.sh
```
Comment (justinsb) [inline Makefile]: I think I would prefer a verify-boilerplate target (and keeping the done message, but I'll just send a PR)

**Comment by justinsb** [comment]
LGTM

File: Makefile (hunk 1 above)
Comment (justinsb) [inline Makefile]: #1047

**Pull request merged**